content/zh/docs/reference/config/networking/virtual-service/index.html

```diff
@@ -120,8 +120,8 @@ <h2 id="CorsPolicy">CorsPolicy</h2>
         host: ratings.prod.svc.cluster.local
         subset: v1
     corsPolicy:
-      allowOrigin:
```

Comment:
https://github.com/istio/istio/blob/ff7aac89141d234438cb8da7d3701822d19683b7/pilot/pkg/networking/core/v1alpha3/route/route.go#L786-L800

`AllowOrigin` still works and has no problems